cumulusci/tasks/salesforce/enable_prediction.py

```diff
@@ -0,0 +1,49 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.core.exceptions import SalesforceException
+from simple_salesforce.exceptions import SalesforceError
+
+
+class EnablePrediction(BaseSalesforceApiTask):
+    task_docs = """
+    cci task run enable_prediction --org dev -o developer_name mlpd__Example_Prediction_v0
+    """
+
+    task_options = {
+        "developer_name": {
+            "description": "Developer name of the MLPredictionDefinition.",
+            "required": True,
+        }
+    }
+
+    def _run_task(self):
+        self._enable_prediction()
+
+    def _enable_prediction(self):
+        try:
+            ml_prediction_definition_id = self._get_ml_prediction_definition_id()
+
+            metadata = self.tooling._call_salesforce(
```

Comment:
We have an emerging pattern for nice error messages from JSON calls:

https://github.com/SFDO-Tooling/CumulusCI/search?q=safe_json_from_response

Reply:
Addressed in 34947f727b18b529bad015bc656f82af045060ec